src/expr/src/scalar/func/impls/string.rs

```diff
@@ -205,6 +205,23 @@ sqlfunc!(
     }
 );
 
+sqlfunc!(
+    #[sqlname = "try_parse_monotonic_iso8601_timestamp"]
+    // TODO: Pretty sure this preserves uniqueness, but not 100%.
+    //
+    // Ironically, even though this has "monotonic" in the name, it's not quite
+    // eligible for `#[is_monotone = true]` because any input could also be
+    // mapped to null. So, handle it via SpecialUnary in the interpreter.
+    fn try_parse_monotonic_iso8601_timestamp<'a>(
+        a: &'a str,
+    ) -> Option<CheckedTimestamp<NaiveDateTime>> {
+        let ts = format::try_parse_monotonic_iso8601_timestamp(a)?;
+        let ts = CheckedTimestamp::from_timestamplike(ts)
+            .expect("monotonic_iso8601 range is a subset of CheckedTimestamp domain");
+        Some(ts)
+    }
+);
```

Comment:
This function is not covered by any test case. Can we add a test case to cover it?

(Feel free to ignore this comment if you feel it is not sensible to do so.)

Reply:
yeah, just confirmed that test/sqllogictest/try_parse_monotonic_iso8601_timestamp.slt does appear in "Fast SQL logic tests 2" in the coverage build that you kicked off. the coverage build doesn't seem to be complaining about the `format::try_parse_monotonic_iso8601_timestamp` that this calls (which contains all the real logic) so it's also possible that my mental model of sql usage of this function going through here is flawed somehow (this is not my normal area of the codebase)

Reply:
Yes, you are absolutely right... I guess the coverage computation is incorrect here, maybe because of the macro...